src/main/java/com/offz/spigot/staminaclimb/ClimbingAxe/ClimbingAxeProperties.java

```diff
@@ -0,0 +1,24 @@
+package com.offz.spigot.staminaclimb.ClimbingAxe;
+
+import org.bukkit.ChatColor;
+import org.bukkit.Material;
+import org.bukkit.block.Block;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.block.BlockBreakEvent;
+
+public class ClimbingAxeProperties implements Listener {
+    @EventHandler
+    public void onBlockBreakWpick(BlockBreakEvent e) {
+        Player p = e.getPlayer();
+        Block block = e.getBlock();
+
+        if (p.getInventory().getItemInMainHand().getType() == Material.IRON_PICKAXE && p.getInventory().getItemInMainHand().getItemMeta().getDisplayName().equals(ChatColor.LIGHT_PURPLE + "Climbing Axe")) {
```

Comment:
Maybe instead of limiting it to one item we could make the item expand the range you can climb (decrease how fast it drains)? Or perhaps different items would allow you to climb on different surfaces. We should just verify what we'd want the system to do, I do agree that having it on for bare hands makes it a little annoying when you're just trying to interact with things, so there might be reason to limit it to the item.